### PR TITLE
🪟🧪 [Experiment] Move OAuth to top of signup page

### PR DESCRIPTION
**Pull request by letiescanciano**
Hypothesis: We know that 50% of our signups use Google OAuth. We can test if bringing it higher on the page increases the signup conversion rate even more. 

Goal: 2% increase of conversion rate in Signup Page.

[Experiment](https://airtable.com/appIuY0uKPVnk8TWT/tbl2SxXnUwf6fVCWS/viww5hSnXY1UUWb6w/rec7XC3rEQDr2MQ09?blocks=hide)

Demo: https://www.loom.com/share/b10dfc5249e244498df85338b27b76d9

## What
Changing the position of the OAuth block on Signup depending on FF value

**Before**
Screen Shot 2022-11-03 at 11 02 08 AM

**After**
Screen Shot 2022-11-03 at 11 01 07 AM

### CODE REVIEW

File: airbyte-webapp/src/packages/cloud/views/auth/OAuthLogin/OAuthLogin.tsx [hunk 1]
```diff
@@ -100,9 +100,6 @@ export const OAuthLogin: React.FC<OAuthLoginProps> = ({ isSignUpPage }) => {
 
   return (
     <div>
-      <div className={styles.separator}>
```
Comment (letiescanciano) [inline airbyte-webapp/src/packages/cloud/views/auth/OAuthLogin/OAuthLogin.tsx:103]: Extracted as `<Separator />`

File: airbyte-webapp/src/packages/cloud/views/auth/SignupPage/SignupPage.module.scss [hunk 2]
```diff
@@ -1,6 +1,12 @@
 @use "../../../../../scss/colors";
 @use "../../../../../scss/variables";
 
+.container {
```
Comment (letiescanciano) [inline airbyte-webapp/src/packages/cloud/views/auth/SignupPage/SignupPage.module.scss:4]: Adding styles for Signup Page Container, as it should be responsible of positioning elements on it.

**Comment by timroes** [comment]
Question (and maybe that's worth putting into the PR description for further experiments in general): What are we trying to test here, i.e. what is the goal of the experiment to figure out? Higher sign-up rates? More percentage of OAuth sign-ups? I'm mostly trying to understand, why we run it as an exeperiment in the first place, instead of simply changing the order without a flag.

**Comment by timroes** [comment]
The separator is lost on the login page with this change:

screenshot-20221107-121108

**Comment by letiescanciano** [comment]
> Question (and maybe that's worth putting into the PR description for further experiments in general): What are we trying to test here, i.e. what is the goal of the experiment to figure out? Higher sign-up rates? More percentage of OAuth sign-ups? I'm mostly trying to understand, why we run it as an exeperiment in the first place, instead of simply changing the order without a flag.

Updated! Agreed that including the goal + hypothesis of the experiment will be helpful.
In this case, I feel like having the FF + Experiment is not a lot of work (and easy to remove in the future) and will allow us to easily change the config from LD and see the difference  between the variant and the baseline. (Also easy experiment to keep evaluating LD Experimentation feature)

**Comment by letiescanciano** [comment]
> The separator is lost on the login page with this change:
> 
> screenshot-20221107-121108

Updated! 
Screen Shot 2022-11-07 at 1 25 33 PM

**Review by timroes — APPROVED** [review]
Code LGTM, have tested locally that the sign-up page and login page are working as expected, with both positions.

**Pull request merged**